library/general/allowedrepos/suite.yaml

```diff
@@ -9,7 +9,23 @@ tests:
     object: samples/repo-must-be-openpolicyagent/example_allowed.yaml
     assertions:
     - violations: no
-  - name: example-disallowed
-    object: samples/repo-must-be-openpolicyagent/example_disallowed.yaml
+  - name: container-disallowed
+    object: samples/repo-must-be-openpolicyagent/disallowed_container.yaml
     assertions:
     - violations: yes
+      message: container
+  - name: initcontainer-disallowed
+    object: samples/repo-must-be-openpolicyagent/disallowed_initcontainer.yaml
+    assertions:
+    - violations: 1
+      message: initContainer
+    - violations: 0
```

Comment:
do we want to make violations consistent? i see some `yes` and `no` in the other suites

Reply:
I wanted to have working examples I can point to from documentation, so I made tests use the full range of violations.